### PR TITLE
View transitions: Add sections on video and entry/exit transitions

### PR DESCRIPTION
**Pull request by jakearchibald**
Adding some more patterns.

### CODE REVIEW

File: site/en/docs/web-platform/view-transitions/index.md [hunk 1]
```diff
@@ -375,6 +383,75 @@ The thumbnail now transitions into the main image. Even though they're conceptua
...
+::view-transition-old(side-bar):only-child {
+  animation: 150ms cubic-bezier(0.4, 0, 1, 1) both fade-out,
+    300ms cubic-bezier(0.4, 0, 0.2, 1) both slide-to-right;
+}
+```
+
+In this case, there's no specific transition for when the side-bar is present in both states, since the default is perfect.
+
+{% Aside %}
+Support for `:only-child` on transition pseudo-elements was added in Chrome 111.
```
Comment (khushalsagar) [inline site/en/docs/web-platform/view-transitions/index.md:452]:
```suggestion
Support for `:only-child` on transition pseudo-elements was added in Chrome 110.
```

**Review by khushalsagar — COMMENTED** [review]
Can't hit approve but LGTM. :)

File: site/en/docs/web-platform/view-transitions/index.md [hunk 2]
```diff
@@ -375,6 +383,75 @@ The thumbnail now transitions into the main image. Even though they're conceptua
...
+    src="video/CZmpGM8Eo1dFe0KNhEO9SGO8Ok23/uXSSraCaMIvu7nbWv0XS.mp4",
+    class="enter-exit-video",
+    loop="true",
+    muted="true",
+    controls="true"
+  %}
+  <figcaption>Entering and exiting sidebar. <a href="https://simple-set-demos.glitch.me/enter-exit-sidebar/">Minimal demo</a>. <a href="https://glitch.com/edit/#!/simple-set-demos?path=enter-exit-sidebar%2Fstyles.css%3A1%3A0">Source</a>.</figcaption>
+</figure>
+
+The side-bar is part of the transition:
```
Comment (rachelandrew) [inline site/en/docs/web-platform/view-transitions/index.md:408]:
```suggestion
The sidebar is part of the transition:
```

File: site/en/docs/web-platform/view-transitions/index.md [hunk 3]
```diff
@@ -375,6 +383,75 @@ The thumbnail now transitions into the main image. Even though they're conceptua
...
+
+```css
+.side-bar {
+  view-transition-name: side-bar;
+  contain: layout;
+}
+```
+
+But, unlike the header in the previous example, the side-bar doesn't appear on all pages. If both states have the side-bar, the transition pseudo-elements look like this:
```
Comment (rachelandrew) [inline site/en/docs/web-platform/view-transitions/index.md:417]: I started changing side-bar to sidebar, but I realise you have that in your code and also further down refer to image-pair.

I think if you are referring to the name as code then use code font.

**Review by rachelandrew — CHANGES_REQUESTED** [review]

File: site/en/docs/web-platform/view-transitions/index.md (hunk 3 above)
Comment (jakearchibald) [reply site/en/docs/web-platform/view-transitions/index.md:417]: I'll make it 'sidebar' in the code too.

**Review by rachelandrew — APPROVED** [review]

**Pull request merged**